profiles/models.py

```diff
@@ -35,29 +38,56 @@
 
 
 class Status(models.Model):
-    code = models.CharField(max_length=3, unique=True)
+    status = models.CharField(max_length=3)
     description = models.CharField(max_length=255)
 
     class Meta:
         verbose_name_plural = "statuses"
 
     def __str__(self):
-        return f"{self.code} - {self.description}"
+        return f"{self.status} - {self.description}"
 
 
 class Ape(models.Model):
     name = models.CharField(max_length=100)
     age = models.PositiveIntegerField()
-    weight = models.DecimalField(max_digits=5, decimal_places=2)
     sex = models.CharField(max_length=1, choices=SEX_CHOICES, default="U")
     profile_picture = models.ImageField(
         upload_to="media/ape_profiles/", null=True, blank=True
     )
+
+    def create_health_record(
+        self, weight=None, swelling=None, stool_type=None, status=None
+    ):
+        health_record = HealthRecord.objects.create(
+            ape=self,
+            weight=weight if weight is not None else self.weight,
+            swelling=swelling if swelling is not None else self.swelling,
+            stool_type=stool_type if stool_type is not None else self.stool_type,
+            status=status if status is not None else self.status,
```

Comment:
These can all be simplified using boolean short-circuiting. Combining the options with `or` will pick the first truthy value. This would reject _any_ falsy value (`None`, empty strings, zeroes, etc) which seems fine here.
```suggestion
            weight=weight or self.weight,
            swelling=swelling or self.swelling,
            stool_type=stool_type or self.stool_type,
            status=status or self.status,
```